src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -511,6 +511,12 @@ def _do_upgrade(self):
                     })
                 return
 
+            # complete mon upgrade?
+            if daemon_type == 'mon':
+                if not self.mgr.get("have_local_config_map"):
+                    logger.info('Upgrade: Restarting mgr now that mons are runnig pacific')
```

Comment:
I seem to recall we had some issues in the past due to which the mgr needed to be upgraded first, wondering if that doesn't hold any longer.